### ISSUE TITLE
Sometimes you pass out before you enter crit and you can't even commit suicide from that state

### ISSUE THREAD

**Issue by Exxion** [d3athrow/vgstation13#7959]
You can't commit suicide while unconscious and you can't succumb unless you're in crit, so if you're unconscious and in healing limbo, you're fucked.

Furthermore, succumb doesn't work for dionae in sufficiently-lit areas. If you are in crit but not dead from burn damage, you can't do fucking shit because you regen the suffocation damage before you die but you can't regen the burn damage to pop out of crit.
In other words, lethal turrets are basically guaranteed to put you in this state.

**Comment by Shadowmech88** [comment]
I didn't think succumb had anything to do with suffocation damage. I thought it just outright killed the mob as long as they were in crit.

**Comment by Exxion** [comment]
Apparently not
I had this happen three times in the same round once as a diona
I ahelped it and the first two times the badmin rejuvenated me and the third time I just asked to be gibbed

**Comment by gbasood** [comment]
Did you try whispering to trigger last words?

**Comment by Exxion** [comment]
I think so, but I don't remember
But I'm pretty sure that does the same thing as succumb anyway
